src/components/review/ProposalReviewContent.tsx

```diff
@@ -74,21 +90,59 @@ const ProposalReviewContent: React.FC<ProposalReviewContentProps> = ({
     />
   );
 
+  const assignAnotherReviewerView = (proposal: Proposal) => {
+    if (proposal.technicalReview?.submitted) {
+      return null;
+    }
+
+    return (
+      <Paper elevation={1} className={classes.reassignContainer}>
+        <Typography variant="h6" gutterBottom>
```

Comment:
The use of a `h6` header here will cause the page to skip heading levels, which is bad for accessibility.
However, I'm working on [an overhaul of the heading levels](https://github.com/UserOfficeProject/stfc-user-office-project/issues/117) throughout the app, so I can pick this up when I make my PR for that.
I just mention for awareness.